src/App.css

```diff
@@ -3091,3 +3092,105 @@ code, .code {
   transform-origin: 0 0; /* see note above */
 }
 .ripple_loader div { box-sizing: content-box; }
+
+.advance-setting-wrapper {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 20px;
+}
+
+p.advance-setting-label {
+  margin-bottom: 5px;
+  line-height: 1.2;
+}
+
+p.advance-setting-description {
+  font-size: 90%;
+}
+
+.advance-setting-link {
+  display: flex;
+  position: relative;
+  transition: opacity 100ms ease-in-out;
+}
+
+a.advance-setting-link { color: #000; }
+.dark a.advance-setting-link { color: #fff; }
+
+.advance-setting-chevron {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  position: relative;
+}
+
+.advance-setting-section {
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+  max-width: 34em;
+}
+
+.chevron-up {
```

Comment:
It would be better to give this a more general name like `.rotate180` since it is not limited to just chevrons. Or add it inline since we're moving towards inline styles eventually.

Reply:
Done